src/components/with-validator/redux/saga.ts

```diff
@@ -60,18 +60,29 @@ function* validateDataGraphRequested({
       );
     }
   } catch (e) {
-    const { message, response } = e as AxiosError;
+    if (e.isAxiosError) {
+      const { message, response } = e as AxiosError;
 
-    if (response?.status === 500) {
+      if (response?.status === 500) {
+        yield put(
+          actions.validateDataGraphFailed(
+            'Noe gikk galt. Vennligst prøv igjen senere.'
+          )
+        );
+      } else {
+        yield put(
+          actions.validateDataGraphFailed(response?.data.detail ?? message)
+        );
+      }
+    } else {
```

Comment:
I stedet for else, er det sikkert mulig å sjekke instance of error for akkurat denne feilmeldingen, men ok for nå.